diff --git a/node_modules/@stackflow/plugin-basic-ui/src/components/AppBar.css.ts b/node_modules/@stackflow/plugin-basic-ui/src/components/AppBar.css.ts
index 6078901..3a68dbc 100644
--- a/node_modules/@stackflow/plugin-basic-ui/src/components/AppBar.css.ts
+++ b/node_modules/@stackflow/plugin-basic-ui/src/components/AppBar.css.ts
@@ -146,7 +146,7 @@ export const left = style([
   f.fullHeight,
   appBarMinHeight,
   {
-    padding: "0 0.5rem",
+    padding: "0 0 0 0",
     ":empty": {
       display: "none",
     },
